Gems/LandscapeCanvas/Code/Source/Editor/MainWindow.cpp

```diff
@@ -2500,6 +2506,24 @@ namespace LandscapeCanvasEditor
         }
     }
 
+    void MainWindow::OnPrefabFocusChanged()
+    {
+        // Make sure to close any open graphs that aren't currently in p refab focus
```

Comment:
nit: _p refab_

Reply:
Whoops thanks will fix! And yes I agree about coming up with a more elegant experience after release.